### ISSUE TITLE
Test for Accessibility 

### ISSUE THREAD

**Issue by rwilliams659** [rwilliams659/overlook-app#48]
Make any changes necessary so the following are fulfilled: 

- Accessibility audits should be at 100%
- A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
- Application should be fully responsive for mobile - tablet - desktop
